addons/l10n_mn/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import chart_template
+from . import res_config_settings
```

Comment:
Hello, 

Thank you for your advice. This all about generate_equity_changes_journals things were used for "Statement of Equity changes" named financial report that I was proposed to add into l10n_mn_reports module. But since this report has removed from l10n_mn_reports, we dont need those generate_equity_changes_journals  things in l10n_mn module anymore.

So I commited my changes.